packages/ui/src/components/Banner/index.tsx

```diff
@@ -0,0 +1,119 @@
+import styled from '@emotion/styled'
+import type { ComponentProps, ReactNode } from 'react'
+import { ButtonV2 } from '../ButtonV2'
+import { Icon } from '../Icon'
+import { Link } from '../Link'
+import { Stack } from '../Stack'
+import { Text } from '../Text'
+
+type Type = 'info' | 'promotional'
+type Size = 'small' | 'medium'
+
+const Container = styled('div', {
+  shouldForwardProp: prop => !['type', 'size'].includes(prop),
+})<{ type: Type; size: Size }>`
+  padding: ${({ theme, size }) => theme.space[size === 'small' ? '2' : '3']};
+  border-radius: ${({ theme }) => theme.radii.large};
+  /* TODO set a gradient instead of a hard color */
```

Comment:
we don't have gradient inside our theme ?

Reply:
https://github.com/scaleway/scaleway-ui/pull/2406

Reply:
This PR was created before the one that adds the gradient, I'll wait for it to be merged to update and use the correct gradient

Reply:
Done @philibea